src/view.tree.js

```diff
@@ -121,9 +121,13 @@ class TreeView {
 
     // refocus after complete so that keyboard navigation works, fix #158
     const refocusAfterCompletion = () => {
-      $(document).one('pjax:success page:load', () => {
-        this.$jstree.get_container().focus()
-      })
+      const refocus = () => this.$jstree.get_container().focus();
+      $(document)
+        .one('pjax:success page:load', () => { // happens when navigating between files
```

Comment:
Is there a reason you can't just add the scroll event to the event string code that already exists? Multiple events are separated by spaces. 🤔 

```js
$(document).one('pjax:success page:load scroll', () => {
```

Reply:
oh, good point!